core/classifiers/CodeClassifier/CodeClassifier.py

```diff
@@ -541,11 +541,22 @@ def to_dict(self):
             dict. A dictionary representation of classifier referred as
             'classifier_data'. This data is used for prediction.
         """
+        # The fingerprint consist of fingerprint components where each
```

Comment:
No, sorry, not here. I meant line 506 (with the correct types).

Reply:
Done.